### PR TITLE
Email template updates

### PR DESCRIPTION
**Pull request by jcontonio**
Updates email templates

### CODE REVIEW

File: api/src/templates/email/mjml/payment-failed.mjml [hunk 1]
```diff
@@ -63,49 +63,22 @@
...
           <h1>Payment Failed to Process</h1>
         </mj-text>
-        <mj-image src="https://www.portwaycontent.com/portway-marketing-assets/app-emails/hero-image.png" />
       </mj-hero>
     </mj-wrapper>
     <mj-section>
       <mj-column>
         <mj-text align="center">
-          <p>Your payment for Portway has failed to process. [TODO: Insert text about how long the account will remain active, steps to take]</p>
+          <p><%= message %></p>
```
Comment (dheniges) [inline api/src/templates/email/mjml/payment-failed.mjml]: this is `textBody` in the renderer

File: api/src/templates/email/mjml/password-change.mjml [hunk 2]
```diff
@@ -63,53 +63,23 @@
         <mj-text align="center">
           <h1>Password Changed</h1>
         </mj-text>
-        <mj-image src="https://www.portwaycontent.com/portway-marketing-assets/app-emails/hero-image.png" />
       </mj-hero>
     </mj-wrapper>
     <mj-section>
       <mj-column>
         <mj-text align="center">
           <p><%= bodyText %></p>
```
Comment (dheniges) [inline api/src/templates/email/mjml/password-change.mjml]: this was renamed `textBody` as well in the render function

**Review by dheniges — CHANGES_REQUESTED** [review]
Some of the templates error out because they're not getting the right variables.

Should also run `npm run compile-email` to push the mjml updates to the html email templates. I have a PR with a little info on this

File: api/src/templates/email/mjml/password-change.mjml [hunk 3]
```diff
@@ -63,53 +63,23 @@
...
       </mj-hero>
     </mj-wrapper>
     <mj-section>
       <mj-column>
         <mj-text align="center">
-          <p><%= bodyText %></p>
-          <p>Use the following link to complete your registration:</p>
+          <p><%= textBody %></p>
         </mj-text>
+          <mj-button href="<% linkUrl %>">Complete your password change</mj-button>
```
Comment (dheniges) [inline api/src/templates/email/mjml/password-change.mjml]: this email is notifying you that your password was changed, it does not require you to click anything in the email. Also that `linkUrl` is undefined. I think this button can just be deleted

**Review by dheniges — CHANGES_REQUESTED** [review]

File: api/src/templates/email/mjml/password-change.mjml (hunk 3 above)
Comment (jcontonio) [reply api/src/templates/email/mjml/password-change.mjml]: Oh woops

**Review by dheniges — APPROVED** [review]

**Pull request merged**